### ISSUE TITLE
Is there a scala guide? It's not referenced in /guides/

### ISSUE THREAD

**Issue by emmanuelbernard** [quarkusio/quarkus#5178]
@gsmet

**Comment by gsmet** [comment]
I don't think so. I can try to ping the guy who initially contributed it and see if he's interested to provide a guide + quickstart.

**Comment by gsmet** [comment]
We should create a simple Scala project here:
https://github.com/quarkusio/quarkus-quickstarts

And have a guide similar to https://quarkus.io/guides/kotlin , at least for the first part of the guide. I don't think we need to dig too much into Gradle for now.

We have an IT for Scala here https://github.com/quarkusio/quarkus/tree/master/integration-tests/scala which can give some ideas.

The dev mode should work with Scala so that's something that needs to be in the guide.

**Comment by carlosedp** [comment]
I wrote a sample Quarkus / Scala 3 app at https://github.com/carlosedp/scala3-quarkus-quickstart.

I think once https://github.com/quarkiverse/quarkus-scala3 starts publishing it's latest version, I can look into writing a Scala 3 guide as a companion to my code. Then, submit the code to the quickstarts repo.

**Comment by radcortez** [comment]
The Scala extension is deprecated and will be removed with https://github.com/quarkusio/quarkus/pull/55094
